views/user/profile.ejs

```diff
@@ -22,12 +23,35 @@
   </div>
   
   <div class="app-lists">
-    
+  
     <div id="your-apps" class="app-list active">
       <div class="d-flex flex-row">
-        <p class="number-of-apps">0 </p>
+        <p class="number-of-apps"><%= data.length %></p>
         <p class="text-in-app-list">Wonderful! Those are sustainable apps that you have added.</p>
       </div>
+    
+      <div class="row" id="app-card">
+        <% data.forEach(app => { %>
+        <div class="col col-lg-4 col-md-6 col-12" onclick="location.href='/app/<%= app._id %>';">
```

Comment:
```suggestion
        <div class="col col-lg-6 col-md-6 col-sm-12" onclick="location.href='/app/<%= app._id %>';">
```

Reply:
this will fix the responsive i think

Reply:
oh no, the problem was different but i tried to change those first, and forgot to change them back as they were. thanks! :)